dashboard/static/js/controllers.js

```diff
@@ -1,8 +1,10 @@
 visApp.controller('mainController', ['$scope', 'Get','$location', '$anchorScroll', function($scope, Get,$location,$anchorScroll) {
-
-   function dataCall(call){
-        var dataFile = '/api/v1/courses/'+dashboard.course_id+'/assignments?percent='+$scope.percent_filter;
-        Get.getData(dataFile).then(function(data) {
+    function dataCall(call, default_value){
```

Comment:
rename to something like "resetToDefault"?

Reply:
This function is actually pulling the data needed for visualization and I am passing the default value as one of the parameters if available. So the name suggested wouldn't be appropriate